pip/pip-346.md

```diff
@@ -0,0 +1,39 @@
+
+# PIP-346: Add a simplified configuration file for Pulsar
+
+# Background knowledge
+
+Nowadays, `broker.conf` contains almost all the configurations of the broker. It's hard to manage the configurations in a single file, which reach almost **500** configurations. 
+
+However, we will only modify a few configurations in the daily work. It's horrible for a beginner to start to learn pulsar, and it's also a potentially error-prone process for advanced users to maintain such a large configuration file.
+
+We can learn from zookeeper that it uses a simplified configuration file, which only contains the configurations that are frequently modified and hides all the configurations that are rarely modified and advanced.
+
+# Goals
+
+## In Scope
+
+Add a simplified configuration file for Pulsar, which only contains the configurations that are frequently modified.
+
+
+# Detailed Design
+
+## Design & Implementation Details
+
+- rename current `broker.conf` to `complete_broker.conf`
+- add a new `broker.conf` which only contains the configurations that are frequently modified
```

Comment:
How do we confirm which configurations need to be modified frequently?

Reply:
I think it depends on the experience.

Reply:
We can vote the configurations in the implementation pr.
It is a process of compression of operation and maintenance experience, which is valuable in my view.

Reply:
I think this is a good discussion starting point, though the precise list and modality should be eventually part of the PIP that is voted on.